music_this_week_app/templates/music_this_week/setup.html

```diff
@@ -12,23 +12,75 @@
         <p><a href="{{ profile }}" target="_blank">{{ username }}</a>&nbsp; &nbsp;&nbsp;<a href="/login/?retry=true">Switch Users</a></p>
         <p><span style="text-decoration: underline;">Search Arguments:</span></p>
         <form action="/search" id="form">
-            <p><strong>Location:</strong> <input id = "loc_field" type="text" name="location" value="San Francisco"></p>
+            <p id = 'location_options'><strong>Location:</strong> <input id = "loc_field" type="text" name="location" value="{{location}}"> 
+               <input type = "checkbox" id = "latlng_checkbox" name = "latlng_checkbox" onchange = "clickCheckBox.call(this)"> Use my current location </input></p>
```

Comment:
Why do you send the `this` argument to `clickCheckBox.call()`? It doesn't seem to be an input argument to the function?

The button (slightly below this) has the following `onchange` field: `buttonClick(this.form)`, but that function actually uses the form object (to submit the form). I'm not sure that any arguments are needed here. Also what's the difference between just calling the function like `clickCheckBox()` as opposed to `clickCheckBox.call()`?